src/styles/partials/components/_model.scss

```diff
@@ -0,0 +1,86 @@
+.dg_model {
+  width: auto;
+  height: auto;
+  background: $black;
+  opacity: 0.95;
+
+  .dg_section {
+    border: 1px solid $blue-lightest;
```

Comment:
Recommend using SCSS variable for 1px. `$border-width-small`.

Reply:
done